### PR TITLE
Moons of Peril Draft

### PR DESCRIPTION
**Pull request by Jin-Jiyunsun**
WIP
Antechamber general:
java_ZAQ3XsskZU
java_x4EFkiIWIw

Eclipse braziers:
java_AZEq18vcZC
java_82RgBK5DnO
They look a little less sad now.

Ancient shrine:
java_BSZBsTchUq
java_p4wm0ymGXN

Blood Moon entrance:
java_6BINSww3ia
java_lq3xztju7S

Blue Moon entrance:
java_t20jKLsKwW
java_gUx89xjiGD

Eclipse Moon entrance:
java_CAv9LzMlp2
java_EHEWMjtM2B

Eclipse Moon arena:
java_pZAhOSCf1g
java_YfVCZGuvfR

Giants Den:
java_8hmndlrxRc
java_DAr3ZLbMQ4

Cam Torum stuff:
java_CvDDxgSe5u
java_mUdKgYXo5h

Closes: #410

### CODE REVIEW

File: src/main/resources/rs117/hd/scene/model_overrides.json [hunk 1]
```diff
@@ -16322,24 +16329,333 @@
...
+      51296,
+      51297,
+      51298,
+      51299
+    ],
+    "uvType": "MODEL_XZ",
+    "uvScale": 0.75
+  },
+  {
+    "description": "Perilous Moons Ancient Shrine stone chips, Hide them they are ugly here",
```
Comment (aHooder) [inline src/main/resources/rs117/hd/scene/model_overrides.json:16501]: This presumably needs to be area-specific, if you want to hide these only in some cases

**Review by aHooder — COMMENTED** [review]
I took the liberty to update this PR, hope that's fine :slightly_smiling_face:
What's left to do for this PR in your view?

File: src/main/resources/rs117/hd/scene/model_overrides.json (hunk 1 above)
Comment (aHooder) [reply src/main/resources/rs117/hd/scene/model_overrides.json:16501]: Also, forgot to mention, it's better to use `"hide": true` than a transparent material.

**Comment by aHooder** [comment]
I've done a run of perilous moons to have a look, and almost everything seems pretty solid :+1:. I'm on the fence about attaching lights to all of the tiny fires making up those temporary flame walls, since that's *a lot* of lights. These mushroom lights are a bit too bright or too short radius too (and they need a better name ofc):
image

File: src/main/resources/rs117/hd/scene/model_overrides.json (hunk 1 above)
Comment (Jin-Jiyunsun) [reply src/main/resources/rs117/hd/scene/model_overrides.json:16501]: I can resolve these tonight

**Comment by Jin-Jiyunsun** [comment]
> I took the liberty to update this PR, hope that's fine 🙂 What's left to do for this PR in your view?

Of course! This was right before I took my month long break so I kinda forgot what I had left to do, I think it was something something to do with tie overrides in the entrance hall with the big D20 since the floor looks like tiles and it currently is just dirt, I think it was pretty much ready for merging besides that and the couple issues you brought up

**Comment by Jin-Jiyunsun** [comment]
> I've done a run of perilous moons to have a look, and almost everything seems pretty solid 👍. I'm on the fence about attaching lights to all of the tiny fires making up those temporary flame walls, since that's _a lot_ of lights. These mushroom lights are a bit too bright or too short radius too (and they need a better name ofc): image

Ah and regarding the lights on the fires, yeah it is a lot but it does make it look quite a bit better imo, is it possible to define world positions for lights and only have them show if a particular object is on screen?

File: src/main/resources/rs117/hd/scene/model_overrides.json (hunk 1 above)
Comment (Jin-Jiyunsun) [reply src/main/resources/rs117/hd/scene/model_overrides.json:16501]: This should be fixed.

**Comment by Jin-Jiyunsun** [comment]
This should be ready for review, maybe do another run through and double check the areas you pointed out before? As for the lights on the long lines of fires, if you deem them too excessive just snip them out before merging and we can revist it later.

**Pull request merged**